packages/terra-application-layout/src/header/ApplicationHeader.module.scss

```diff
@@ -22,6 +22,10 @@
     }
   }
 
+  .application-navbar-with-icons {
+    height: var(--terra-application-layout-navbar-with-icons-height, 5.142857143rem);
```

Comment:
I think you'll want to add some code for a different height when the layout is in a narrow "mobile" screen. Something like

```scss
@import '~terra-breakpoints/lib/media-queries';

.application-navbar-with-icons {
  height: var(--terra-application-layout-navbar-with-icons-mobile-height, 3.5714285714rem);

  @include terra-mq-large-up {
    height: var(--terra-application-layout-navbar-with-icons-height, 5.142857143rem);
  }
}
```

You'll need to add [terra-breakpoints](https://engineering.cerner.com/terra-ui/#/components/terra-breakpoints/breakpoints/about) as a dependency to terra-application-layout to pull in the media query mixin.